### PR TITLE
Use historic quantiles with daily data for viz dates

### PR DESCRIPTION
**Pull request by lindsayplatt**
Lots of cleaning/reorganizing, but a few main things:

1. Remove climate response gage code
2. Adjust NWIS fetch fxns in `1_fetch` to use bounding boxes to query NWIS so that we can easily change the focus region in the future
3. Add two targets that take daily data for the time period and figure out the quantiles + categorize into very low to very high.

Here is a quick look at the distribution and a map (didn't implement into pipeline for now, just illustrating what data is available for FY20 Q1 and including the code for posterity).

```r
library(ggplot2)
library(readr)
library(scipiper)
library(dplyr)
library(maps)

scmake("2_process/out/gw_data_anomalies.csv")
scmake("1_fetch/out/gw_site_info.csv")
dat <- read_csv("2_process/out/gw_data_anomalies.csv", col_types = 'cDnnc') 
dat_info <- read_csv("1_fetch/out/gw_site_info.csv", col_types = 'cccnn')

ggplot(dat, aes(x = quant_category)) +
  geom_histogram(stat = "count")

dat_one_day <- dat %>% 
  filter(Date == as.Date("2020-10-31")) %>% 
  left_join(dat_info) %>% 
  mutate(color = ifelse(
    quant_category == "Very high",
    yes = "#304965", no = ifelse(
      quant_category == "High",
      yes = "#5ea0d8", no = ifelse(
        quant_category == "Normal",
        yes = "#818a7f", no = ifelse(
          quant_category == "Low",
          yes = "#d29958", no = ifelse(
            quant_category == "Very low",
            yes = "#a02905", no = "black"))))))

par(mar = c(0,0,0,0))
map('usa', fill=TRUE, col = "grey", border = NA)
map('state', col = "#fcf8e8", add=TRUE)
points(dat_one_day$dec_long_va, dat_one_day$dec_lat_va, col = dat_one_day$color, cex = 1, pch = 20)
```

All data in FY20 Q1:
image

Just Halloween: 
image

image

### CODE REVIEW

File: 0_config.yml [hunk 1]
```diff
@@ -26,6 +26,12 @@ targets:
     command: as.Date(I('2020-10-01'))
   viz_end_date:
     command: as.Date(I('2020-12-31'))
+  # This was a useful resource: https://anthonylouisdagostino.com/bounding-boxes-for-all-us-states/
+  # Use bounding box in case we want to do this regionally someday
+  # Must be west longitude, south latitude, east longitude, and then north latitude
+  # Start with CONUS only
+  viz_bounding_box:
+   command: c(I(-124.409591), I(24.523096), I(-66.949895), I(49.384358))
```
Comment (collnell) [inline 0_config.yml:34]: smart!

File: 1_fetch/src/fetch_nwis.R [hunk 2]
```diff
@@ -1,25 +1,82 @@
...
       error = function(e) return()
     ) %>% c(sites)
   }
   
   return(sites)
+  
+}
+
+#' Function to divide the requested bounding box into NWIS query-able cells
+prep_latlon_cells <- function(long_vals, lat_vals, deg_per_slice) {
```
Comment (collnell) [inline 1_fetch/src/fetch_nwis.R:34]: 🥇

**Review by collnell — APPROVED** [review]

File: 1_fetch/src/fetch_nwis.R (hunk 2 above)
Comment (lindsayplatt) [reply 1_fetch/src/fetch_nwis.R:34]: this took more time to do than I care to admit 😀

**Comment by collnell** [comment]
Amazing progress! What a fun site distribution 😺 That is actually quite a high proportion of data points that fall within the "very high" category. I wonder what's going on there

**Comment by lindsayplatt** [comment]
I agree! Will be fun to dig into it more :)

**Pull request merged**